core/server/models/base/index.js

```diff
@@ -394,17 +394,17 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
     },
 
     // Get the user from the options object
-    contextUser: function contextUser(options) {
-        options = options || {};
-        options.context = options.context || {};
+    contextUser: function contextUser(options = {}) {
+        const context = options.context || {};
```

Comment:
_Technically_ this changes the functionality:

```js
const options = {whatever: 'blah'};
models.Base.Model.prototype.contextUser(options);
assert(options.context); // throws now
```

However I can't see anywhere (or reason) that we rely on that.